tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -118,68 +117,1144 @@ - (void)dealloc {
         ASSERT_EQ_MSG(xamlElement.Get(), nullptr, "Unexpectedly able to reacquire a strong reference to the backing Xaml element.");
     }
 
-    TEST_METHOD(UIButton_TitleColorChanged) {
-        __block auto uxEvent = UXEvent::CreateManual();
+    TEST_METHOD(UIButton_CreateButtonWithDefaultInit) {
+        StrongId<UIButtonWithControlsViewController> buttonVC;
+        buttonVC.attach([[UIButtonWithControlsViewController alloc] init]);
+        UXTestAPI::ViewControllerPresenter testHelper(buttonVC);
+
+        UIButton* buttonToTest = [buttonVC defaultButton];
+
+        WXCControl* xamlControl = rt_dynamic_cast([WXCControl class], [buttonToTest xamlElement]);
+        ASSERT_OBJCNE(xamlControl, nil);
+
+        // Extract UIButton.titleLabel control to verify its visual state
+        WXFrameworkElement* titleLabelXamlElement = [buttonToTest.titleLabel xamlElement];
+        ASSERT_OBJCNE(titleLabelXamlElement, nil);
+
+        WXCTextBlock* textBlock = rt_dynamic_cast([WXCTextBlock class], titleLabelXamlElement);
+        ASSERT_OBJCNE(textBlock, nil);
+
+        // Background color
+        dispatch_sync(dispatch_get_main_queue(), ^{
+            WUXMSolidColorBrush* solidBackgroundBrush = rt_dynamic_cast([WUXMSolidColorBrush class], xamlControl.background);
+            UIColor* expectedBackgroundColor = [UIColor colorWithRed:1.0f green:1.0f blue:1.0f alpha:0.0f];
+            EXPECT_TRUE(UXTestAPI::IsRGBAEqual(solidBackgroundBrush, expectedBackgroundColor));
+        });
+
+        // Title color
+        __block auto uxEvent = UXEvent::CreateAuto();
         __block auto xamlSubscriber = std::make_shared<XamlEventSubscription>();
 
-        UIButtonWithControlsViewController* buttonVC = [[UIButtonWithControlsViewController alloc] init];
+        dispatch_async(dispatch_get_main_queue(), ^{
+            // Register RAII event subscription handler
+            xamlSubscriber->Set(textBlock, [WXCTextBlock foregroundProperty], ^(WXDependencyObject* sender, WXDependencyProperty* dp) {
+                WUXMSolidColorBrush* solidBrush = rt_dynamic_cast([WUXMSolidColorBrush class], [sender getValue:dp]);
+
+                // Validation
+                if (UXTestAPI::IsRGBAEqual(solidBrush, [UIColor whiteColor])) {
```

Comment:
>whiteColor [](start = 64, length = 10)

is whiteColor our contract for titleColor for default (normal) state for Default type UIButton? 

I was playing around and seems The ios contract is that normal state title color for normal state for default type of UIButton is default to Tint Color which is set to Blue for UIButton. or at least this is on iOS 10.